composer.json

```diff
@@ -30,8 +30,10 @@
         "symfony/phpunit-bridge": "^5.0",
         "nyholm/symfony-bundle-test": "^1.5",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
-        "doctrine/doctrine-bundle": "^v1.0|^v2.0",
-        "symfony/twig-bundle": "^3.4|^4.0|^5.0"
+        "doctrine/annotations": "^v1.7",
+        "doctrine/cache": "^v1.0",
+        "symfony/twig-bundle": "^3.4|^4.0|^5.0",
+        "symfony/browser-kit": "^3.4|^4.0|^5.0"
```

Comment:
Is this dev requirement needed because of the tests? Can we prevent this from being added?

Reply:
Removed, not sure why I added it

Reply:
Oh, right, the new tests with lowest Symfony version fail without it due to a dependency injection bug, which was only fixed in versions 3.4.19 and 4.1.8 (https://github.com/symfony/symfony/pull/29104)

Reply:
There's some Symfony dependency injection bug which prevents auto-wiring the FrameworkBundle in the older versions and I'm not sure when it was fixed. On the other hand, I'm not sure if I really need the framework bundle to run the tests

Reply:
Got around it by disabling autoconfigure and autowire in `src/Resources/config/services.xml`